guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -113,15 +113,19 @@ endif::[]
 This can be set by the `host_packages` parameter.
 . Optional: Select the *Update packages* option to update all packages on the host upon registration.
 This can be set by the `host_update_packages` parameter.
-. Optional: In the *Repository* field, enter a repository to be added before the registration is performed.
+. In the *Repositories* field, click the *+* symbol.
```

Comment:
```suggestion
. In the *Repositories* field, click *+ Add repositories for registration*.
```
Or even without the "+", as you prefer. I'm assuming that the text is clickable.

Reply:
Applied without `+`. Looks cleaner & more readable this way. Text is clickable.